### PR TITLE
Make parameter name match local variable name and some typos

### PR DESCRIPTION
**Pull request by Vinietskyzilla**

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
I think you did several changes in a single commit, please split them into multiple commits, thanks.

**Comment by Vinietskyzilla** [comment]
I'm sorry but I must say I feel you are being a little picky. I'll do as you asked though.

**Comment by Vinietskyzilla** [comment]
Did you notice that the commit message for 16caea9 has several lines that describe all of the changes?

**Comment by Vinietskyzilla** [comment]
Also do you have a document that describes criteria for mergine pull requests? Maybe I could avoid multiple cycles of fixing my pull requests if I know from the start what the rules are.

**Comment by Vinietskyzilla** [comment]
Sorry for all the complaining, I hope it's better now.

**Comment by PeterDaveHello** [comment]
and I wonder if you can put the coding style commits in another PR?

File: backbone.js/what-is-a-model/index.md [hunk 1]
```diff
@@ -198,9 +193,8 @@ Now that we have saved a new user model, we can retrieve it from the server.   W
 If we instantiate a model with an `id`, Backbone.js will automatically perform a get request to the urlRoot + '/id' (conforming to RESTful conventions)
 
 ```js
-
-// Here we have set the `id` of the model
-var user = new Usermodel({id: 1});
+// Here we have set the id of the model
+var user = new UserModel({id: 1});
```
Comment (PeterDaveHello) [inline backbone.js/what-is-a-model/index.md]: mistake here ...

**Comment by PeterDaveHello** [comment]
About 395bdc92f1e908c4eaa9002261e71a6d69a6d823, I this it's just to emphasize the word, I don't know why should we remove it.

**Comment by PeterDaveHello** [comment]
and I wonder if you can put the coding style commits in another PR?

**Comment by Vinietskyzilla** [comment]
I can't waste any more time fixing up these commits. Take it or leave it.

**Comment by lite20** [comment]
^ +1

**Comment by PeterDaveHello** [comment]
I'll do another commit to fix it.

**Pull request closed without merging**